### DISCUSSION TITLE
neogit.log

### DISCUSSION THREAD

**Discussion by ur4ltz** [NeogitOrg/neogit#254] [Q&A] [answered]
How to disabe debug neogit.log?

**Comment by CKolkey** [comment]
Do you mean how to prevent it from writing to disk?

**Reply by ur4ltz** [reply]
I don't use the plugin now, but I was interested in the ability to disable the log, change the level of detail of the logging. He wrote every cursor movement to the log, I got tired of it and removed the plugin. If this has changed then I will return the plugin.

I'm sorry, this is a google translation.

**Reply by CKolkey** [reply, marked as answer]
No worries. We're no longer logging by default, and you can control the level too. 

https://github.com/NeogitOrg/neogit/blob/master/lua/neogit/logger.lua

**Reply by ten3roberts** [reply]
What a coincidence, just added a section about the controlling env variables to the contributing guide PR
